### PR TITLE
added 'Bind to grammar/schema file' command to Commands.md

### LINKED ISSUE #561: Add `Bind to grammar/schema file` command in docs
In https://github.com/redhat-developer/vscode-xml/blob/master/docs/Commands.md#commands it should be good to add the `Bind to grammar/schema file` command.

### PR DESCRIPTION
**Pull request by AlexXuChen**
Added documentation for 'Bind to grammar/schema file' command in `Commands.md`.

Closes #561

Signed-off-by: Alexander Chen alchen@redhat.com

### CODE REVIEW

File: docs/Commands.md [hunk 1]
```diff
@@ -4,9 +4,15 @@
 
 ![XML Commands](images/Commands/XMLCommands.png)
 
+## Bind to grammar/schema file
+
+This command triggers the [XML Binding Wizard]() for the current file.
```
Comment (AlexXuChen) [inline docs/Commands.md:9]: I think we should document the XML Binding Wizard feature somewhere (unless I missed it) so I can link it here.
Thoughts @angelozerr?

Comment (AlexXuChen) [reply docs/Commands.md:9]: If so, it can also be linked in #562

Comment (angelozerr) [reply docs/Commands.md:9]: Indeed, we should have a new `BindingWithGrammar.md` file referenced in https://github.com/redhat-developer/vscode-xml/tree/master/docs#user-guide.


BindingWithGrammar.md could looks like:

## Binding with  existing grammar

In the following sections, we would like to bind the XML `foo.xml` file:

```xml
<foo>
  <bar />
</foo>
```

with the XSD `foo.xsd` file (in the same folder as foo.xml) :

```xml
<?xml version="1.0" encoding="UTF-8"?>
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
  <xs:element name="foo">
    <xs:complexType>
      <xs:sequence>
        <xs:element name="bar" />
      </xs:sequence>
    </xs:complexType>
  </xs:element>
</xs:schema>
```

### Command

### Codelens

### Quickfix

## Binding with new grammar

Imagine you have the XML `foo.xml` file:

```xml
<foo>
  <bar />
</foo>
```

and you want to generate a grammar and bind it.

### Quickfix

Comment (AlexXuChen) [reply docs/Commands.md:9]: Should I open a separate issue and pr for this new page then?

**Comment by AlexXuChen** [comment]
Need to merge #570 before this change

**Review by datho7561 — APPROVED** [review]
LGTM

**Pull request merged**